speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/DecisionsTests.cpp

```diff
@@ -38,13 +38,14 @@ void InitializeMediator()
 TEST(DecisionsTest, RunInterveneDecisions)
 {
     InitializeMediator();
+    tAllowedActions allowedActions = {true, true, true};
```

Comment:
You should also test with each of these values being false at least once. Perhaps use the combinatorial tests for this as well

Reply:
The decisions are independent as far as I can tell, they're just all tested in the same test. The testing method for true and false also differs, retrieving value first and making sure it is unchanged. So I don't think I combinatorial tests are the best for this. I do agree I should add tests though.

Reply:
You can disregard this, it is actually achievable quite nicely.